src/lib/api/event.recorder.ts

```diff
@@ -262,8 +265,8 @@ export function createListenHandler(
       prebootData.activeNode = undefined;
     }
 
-    // if we are freezing the UI
-    if (eventSelector.freeze) {
+    // if overlay is not disabled and we are freezing the UI
+    if (!(opts.disableOverlay || false) && eventSelector.freeze) {
```

Comment:
`|| false`?

Reply:
Sure, we can omit it here, as we are already setting the default value for `disableOverlay `

Reply:
Done